server/routes/surveyRoutes.js

```diff
@@ -61,13 +35,13 @@ module.exports = app => {
       title,
       subject,
       body,
-      recipients: recipients.split(',').map(email => ({ email: email.trim() })),
+      recipients: surveyService.getRecipientListFromEmailString(recipients),
```

Comment:
I also added a helper method to my service to handle this confusing logic. Putting this code directly in an object definition is gross. Instead I made a nicely named function in my service and I call it here. Because it's nicely named, you know what it does just by looking, `getRecipientListFromEmailString`. It also keeps more business logic in the service layer. Really most of this could be moved to the service, but whatever, this is fine for now.